src/main/java/edu/kis/powp/jobs2d/command/TaskImmutableCommandBuilder.java

```diff
@@ -0,0 +1,32 @@
+package edu.kis.powp.jobs2d.command;
+
+
+public class TaskImmutableCommandBuilder {
```

Comment:
Czy to jest PR z visitorem czy też z tym drugim Panów zadaniem?

Reply:
PR dotyczy głównie A visitor for exceeding canvas (and margin) by a complex command. Pozdrawiamy